agent-inject/agent/annotations_test.go

```diff
@@ -9,14 +9,16 @@ import (
 
 	corev1 "k8s.io/api/core/v1"
 
+	"github.com/hashicorp/vault/sdk/helper/pointerutil"
 	"github.com/mattbaird/jsonpatch"
+	"github.com/stretchr/testify/require"
 )
 
 func TestInitCanSet(t *testing.T) {
 	annotations := make(map[string]string)
 	pod := testPod(annotations)
 
-	err := Init(pod, AgentConfig{"foobar-image", "http://foobar:8200", "test", "test", true, "1000", "100"})
+	err := Init(pod, AgentConfig{"foobar-image", "http://foobar:8200", "test", "test", true, "1000", "100", DefaultAgentRunAsSameUser, DefaultAgentSetSecurityContext})
```

Comment:
It might be time to define this AgentConfig struct outside of the function call.  It's getting rather large.

Reply:
Yep, done in 3c1cf60.